### PR TITLE
Added SQL Indices to Init

### PR DESCRIPTION
**Pull request by j48zheng**
Also made index column primary key.

### CODE REVIEW

File: holoclean/dataengine.py [hunk 1]
```diff
@@ -243,6 +243,8 @@ def _dataframe_to_table(self, spec_table_name, dataframe, append=0):
                     create_table = create_table + "INT,"
                 else:
                     create_table = create_table + "VARCHAR(255),"
+            if 'index' in dataframe.schema.names:
```
Comment (thodrek) [inline holoclean/dataengine.py]: why have we hardcoded the index? What if the user does not specify an index? We still assign an index (i.e., an id) to each tuple.

File: holoclean/utils/pruning.py [hunk 2]
```diff
@@ -493,15 +493,4 @@ def _create_dataframe(self):
 
         self.dataengine.query(query_observed)
 
-        self.dataengine.holo_env.logger.info('The table: ' +
```
Comment (thodrek) [inline holoclean/utils/pruning.py:497]: where did these go? Were they moved before the query is executed?

Comment (j48zheng) [reply holoclean/utils/pruning.py:497]: I felt these were not needed. They don't print the query. They just print that the the table has been created.

File: holoclean/dataengine.py (hunk 1 above)
Comment (j48zheng) [reply holoclean/dataengine.py]: For the current version we assume each dataset has a column called "index" that counts from 1 to the number of tuples

Comment (thodrek) [reply holoclean/dataengine.py]: is this enforced then when we load a dataset? If not it should be added.

File: holoclean/utils/pruning.py (hunk 2 above)
Comment (thodrek) [reply holoclean/utils/pruning.py:497]: Ah ok. This is fine then.

**Review by ah89 — APPROVED** [review]

**Pull request merged**